doctrine.rst

```diff
@@ -638,10 +632,11 @@ Fetching an object back out of the database is even easier. For example,
 suppose you've configured a route to display a specific ``Product`` based
 on its ``id`` value::
 
-    public function showAction($productId)
+    use Doctrine\ORM\EnityManagerInterface;
+
+    public function showAction($productId, EnityManagerInterface $em)
     {
-        $product = $this->getDoctrine()
-            ->getRepository('AppBundle:Product')
+        $product = $em->getRepository('AppBundle:Product')
             ->find($productId);
```

Comment:
[DX] ``$em->find(Product::class, $productId)``? at the end it does the same, so less coding :)